data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1831 @@
+			`	After a few hours, the storm finally subsides, and you both return to your ship. you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
+				accept
+			label bumpyblizzard
+			`	Iplora hesitates for a bit, then explains that you would probably get sick going through the storm now. She suggests you wait in the cave until it is safe for human passage.`
+			`	You insist, and she reluctantly gives in, telling you to stick close to her. She moves slowly, so you have no problem keeping up, but time and again you keep falling over, tripping on the mounds the rocks have created. Another fall, and Iplora reaches her hand out to help you up once again.`
+			choice
+				`	(Take her hand.)`
+				`	(Try and dig through the snow to find something.)`
+					goto snowdig
+			label back
+			`	She helps you up, and you continue on your way down the hills. You get back to the forest, and she starts finding the trail back to the ship. Halfway through, the storm gets progressively weaker.`
+			`	When you two return to your ship. you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
+				accept
+			label snowdig
+			`	You start clawing your way down the snow as Iplore asks you to stop and continue moving toward the ship You continue, your hands getting colder as the snow steals the heat even through your gloves. You hit a hard surface and start clearing from there, only to see it was, in fact, just a boulder.`
+			`	"Satisfied? Return, we must. Come now," You turn to see her offering her hand once again.`
+			choice
+				`	(Take her hand.)`
+					goto back
+				`	(Head to another mound and try digging there.)`
+			label keepdig
+			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, and a faint metallic sound echoes. She takes her hands off of you and lays down, now helping you dig out whatever is there. She stands up once the crate is visible. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`
+			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast, coming from the the top of the hills, hits the ground near the hole, missing you by just about a meter.`
+			`	She yells for you to get down, pulling out her weapon and firing in the direction the shot came from.`
+			choice
```

Comment:
small thing: I noticed you give the evidence in both cases for the choice (taking cover and helping) so maybe put it here, before the `choice`?